src/main/java/org/jmxtrans/embedded/output/CopperEggWriter.java

```diff
@@ -258,12 +258,12 @@ public void write(Iterable<QueryResult> results) {
         List<QueryResult> app_counters = new ArrayList<QueryResult>();
         List<QueryResult> app_sales_counters = new ArrayList<QueryResult>();
 
-        long epochInMillis = 0;
-        String myname =  null;
-        Object myval = null;
-        long thisPID = 0;
-        String tmp = null;
-        String pidHost = null;
+        long epochInMillis;
```

Comment:
Sorry @AymanDF but I understand that the author has chosen to initialize the variables explicitly to zero for "self documentation purpose".